user-interface/src/data-verification/TransferOrderAccordion.tsx

```diff
@@ -90,251 +23,15 @@ export interface TransferOrderAccordionProps {
-
-    const updatedOrderTransfer = updateOrderTransfer(orderTransfer, office, newCaseNumber);
-    setOrderTransfer(updatedOrderTransfer);
-    if (office && newCaseNumber) {
-      isValidOrderTransfer(updatedOrderTransfer).then((valid) => {
-        approveButtonRef.current?.disableButton(!valid);
-      });
-    }
-  }
-
-  function handleCaseInputChange(caseNumber?: string) {
-    if (caseNumber) {
-      setNewCaseNumber(caseNumber);
-    } else {
-      setNewCaseNumber(null);
-      setValidationState(ValidationStates.notValidated);
-      setNewCaseSummary(null);
-      approveButtonRef.current?.disableButton(true);
-      return;
-    }
-
-    const updatedOrderTransfer = updateOrderTransfer(orderTransfer, newCaseDivision, caseNumber);
-    setOrderTransfer(updatedOrderTransfer);
-    if (caseNumber && newCaseDivision) {
-      isValidOrderTransfer(updatedOrderTransfer).then((valid) => {
-        approveButtonRef.current?.disableButton(!valid);
-      });
-    }
-  }
-
-  function handleSuggestedCaseSelection(bCase: CaseSummary) {
-    if (bCase) {
-      const updated = { ...orderTransfer };
-      updated.newCase = bCase;
-      setOrderTransfer(updated);
-      approveButtonRef.current?.disableButton(false);
-    }
-  }
-
-  function confirmOrderApproval(): void {
-    orderTransfer.status = 'approved';
-
-    const updatedOrder: TransferOrder = {
-      ...order,
-      ...orderTransfer,
-    } as TransferOrder;
-
-    api
-      .patch(`/orders/${orderTransfer.id}`, orderTransfer)
-      .then(() => {
-        props.onOrderUpdate(
-          {
-            message: `Transfer of case to ${getCaseNumber(orderTransfer.newCase?.caseId)} in ${
-              orderTransfer.newCase?.courtName
-            } (${orderTransfer.newCase?.courtDivisionName}) was ${orderTransfer.status}.`,
-            type: UswdsAlertStyle.Success,
-            timeOut: 8,
-          },
-          updatedOrder,
-        );
-      })
-      .catch((reason) => {
-        // TODO: make the error message more meaningful
-        props.onOrderUpdate({ message: reason.message, type: UswdsAlertStyle.Error, timeOut: 8 });
-      });
-  }
-
-  function resetState() {
-    setOrderTransfer(getOrderTransferFromOrder(order));
-    setNewCaseSummary(null);
-    setValidationState(ValidationStates.notValidated);
-    if (suggestedCasesRef.current) suggestedCasesRef.current.clearAllCheckboxes();
-    setLoadingCaseSummary(false);
-    approveButtonRef.current?.disableButton(true);
-  }
-
-  function cancelUpdate(): void {
-    courtSelectionRef.current?.clearValue();
-    caseNumberRef.current?.resetValue();
-    resetState();
-  }
-
-  function confirmAction(status: OrderStatus, reason?: string): void {
-    if (status === 'rejected') {
-      approveOrderRejection(reason);
-    } else if (status === 'approved') {
-      confirmOrderApproval();
-    }
-  }
+  // const [activeButtonId, setActiveButtonId] = useState<string>(`suggested-cases-${order.id}`);
```

Comment:
probably want to remove this

Reply:
This is tied to some code below which needs to be assessed as to whether it's needed anymore.

Reply:
Not used. Deleting.